src/sb_pointcloud_processing/src/test_pcl_generator_node.cpp

```diff
@@ -100,20 +105,20 @@ sensor_msgs::PointCloud2 generatePclMessage(bool include_outlier) {
     // Generate a single PointCloud with noise
     pcl::PointCloud<pcl::PointXYZ> pcl;
     LineExtractor::TestUtils::addLineToPointCloud(
-    args, pcl, max_noise_x, max_noise_y);
+    args, pcl, max_noise_x, max_noise_y, false);
```

Comment:
In general, when writing tests we want the results to be "deterministic" (that is, that same every time). if I understand the `same_seed` argument correctly, setting it false means that the line generated will be generated differently each time. One option to make this deterministic might be to add another argument to `addLineToPointCloud` that lets you set the seed used to generate the line, then manually choose a seed and set `same_seed` to `false`.

Reply:
done